addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.js

```diff
@@ -34,5 +34,6 @@ export class AvatarCardPopover extends Component {
 
     onSendClick() {
         this.openChat(this.user.id);
+        this.props.close();
```

Comment:
Watch out starting from `saas-17.1`, There's extended composer `AvatarCardResourcePopover` that overrides this method to call `openChat(record.user_id[0])`. I presume this also requires the same fix.
_(would be nice to not repeat `.close()` again in override)_